zipkin-storage/elasticsearch-http/src/main/java/zipkin/storage/elasticsearch/http/ElasticsearchHttpSpanConsumer.java

```diff
@@ -37,27 +38,45 @@
 
   final ElasticsearchHttpStorage es;
   final IndexNameFormatter indexNameFormatter;
+  private final static int MAX_CACHE_DAYS = 2;
```

Comment:
can make a comment that usually spans will be in the same day, except at midnight. below, that entries correspond to unique service/span combinations. it is probably find to say 1000, but even 10000 should be reasonable as there's not much ram involved (ex if you have 1000 services, you will need a count greater than that for the diversity of span names w/in a service).